meetings/2022-01-26.md

```diff
@@ -0,0 +1,97 @@
+* Matthieu: While I think this is all intersting/important, I'm not sure this is in scope of authorization panel to find a solution to such a (social) problem.
+* Elf: What about we focus on the acl:Control problem? If you give someone access, we cannot control impersonation.
+* Matthieu: That's exactly what I'm talking about; there is no technical way to prevent social engineering (since it is based on trust). We will not *solve* this issue.
+* Elf: You still have a kind of delegated control (controlled impersonation). The notion of acl:Control may mislead the user into thinking that unless this is given/delegated, they cannot share that access.
+* Matthieu: Do you suggest that sharing access to a resource would never happen? You would just share read, write and append access, and not let other people share access further? Wouldn't a colaborative environments be just copies of copies of access?
+* Elf: You enable people to clearly delegate access. In this way, you can have clear (reportable, cf. infra: logs of who does what on whose behalf) chains. This does not have the same misleading aspect. You just need a specified way to do delegation.
+* Matthieu: How would the delegation chain show up in an access token for example?
+* Elf: In short, there would be 2 ways:
+    * Relying on back-channel (see https://github.com/solid/data-interoperability-panel/issues/222). In the end, there are some racing conditions. In the end, the RS would have simple ACL rules on the resource.
+    * Relying on signatures and signatures chains, using pushed claims to the AS: downstream signatures would be on the claim.
+* Matthieu: That doesn't necessarily take away the need for a Control mode. People should be able to set, e.g. rules which issuer could access which resource etc. on a Solid server. This more "standard" way of giving control seems to be a better way. Maybe on the ACL level you still want something specific.
+* Elf: There is an issue on ACP access would be simply access to the ACR itself. Only the AS should be able to set these, so we could configure this access (of the AS) on a higher level in the RS. So there is no need to set access on ACRs (at least not to respond to requests).
+* Matthieu: For me this is more of a data modelling issue, but in the end we are on the same page.
+* Wouter: While I do agree that the questions about legally enforcing policies are not in the scope of the panel, is there a place where we could dive in the boundaries between the technical and legal aspect?
+* Elf: I'm not sure, but I'm also interested.
+* Wouter: Maybe we can look into that.
+* Tom: I've thought about that a lot in the past and the goals in the pannels is to provide something that is privacy compliant. It might be hard to enforce, but at least providing adequate tooling for the willing, that would be great. Maybe a data-protection panel? It might not be an issue on the technical level, but on the user level.
```

Comment:
```suggestion
* Tom: I've thought about that a lot in the past, and the goal in the panels is to provide something that is privacy compliant. It might be hard to enforce, but if we could at least providing adequate tooling for the willing, that would be great. Maybe a data-protection panel? It might not be an issue on the technical level, but on the user level.
```